public/app/screens/main/sections/settings/user-form/user-form.jsx

```diff
@@ -0,0 +1,49 @@
+var React = require('react');
+var Reflux = require('reflux');
+var styleMixin = require('mixins/style-mixin');
+var t = require('tcomb-form');
```

Comment:
Because of this line for example,
```var Form = tForms.form.Form;```
with tcombForm it would be like :
```var Form  = tcombForm.form.Form;```
which is tavtology as for me.